voorhoede/head-start
# feature: Details Block

# Changes

- Adds new Details Block

# Associated issue

Resolves #171 

# How to test

1. Open preview link
2. Navigate to /en/demos/page-partial-layouts/
3. You can also check Dato.

# Checklist
